Integrations/integration-jamf.yml

```diff
@@ -30,213 +30,164 @@ configuration:
+                "BarCode-2": item["bar_code_2"],
+                "Building": item["building"],
+                "BuildingName": item["building_name"],
+                "Department": item["department"],
+                "DepartmentName": item["department_name"],
+                "Email": item["email"],
+                "EmailAddress": item["email_address"],
+                "Position": item["position"],
+                "Username": item["username"],
+                "Realname": item["realname"],
+                "Room": item["room"]}
+                for item in contents["computers"]
+            ]
+
+            demisto.results({ 'Type' : entryTypes['note'], 'ContentsFormat' : formats['json'], 'Contents' : contents, "EntryContext": ec,"HumanReadable": tableToMarkdown("Computers", ec["JAMF"])})
+        except Exception,e:
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Unable to complete command.  Please check credentials and/or proxy settings" })
+            sys.exit(0)
+
+    def jamf_computers_hardware(computer_id):
+        endpoint = "/JSSResource/computers/id/{computer_id}/subset/hardware"
+
+        try:
+            contents = requests.get(base_url + endpoint.format(computer_id=computer_id),auth=auth,headers=HEADERS).json()
+            ec = {}
+            ec["JAMF"] = {
+                "Make": contents["computer"]["hardware"]["make"],
+                "Model": contents["computer"]["hardware"]["model"],
+                "ModelIdentifier": contents["computer"]["hardware"]["model_identifier"],
+                "OSName": contents["computer"]["hardware"]["os_name"],
+                "OSVersion": contents["computer"]["hardware"]["os_version"],
+                "OSBuild": contents["computer"]["hardware"]["os_build"],
+                "MasterPasswordSet": contents["computer"]["hardware"]["master_password_set"],
+                "ActiveDirectoryStatus": contents["computer"]["hardware"]["active_directory_status"],
+                "ServicePack": contents["computer"]["hardware"]["service_pack"],
+                "ProcessorType": contents["computer"]["hardware"]["processor_type"],
+                "ProcessorArchitecture": contents["computer"]["hardware"]["processor_architecture"],
+                "NumberProcessors": contents["computer"]["hardware"]["number_processors"],
+                "NumberCores": contents["computer"]["hardware"]["number_cores"],
+                "TotalRam": contents["computer"]["hardware"]["total_ram"],
             }
-            return res;
-        };
-        return function(obj) {
-            if (obj instanceof Array) {
-                var res = [];
-                for (var j=0; j < obj.length; j++) {
-                    var current = transformSingleObj(obj[j]);
-                    if (current) {
-                        res.push(current);
-                    }
+
+            ec["JAMF"]["Storage"] = [
```

Comment:
Same as above should be `Jamf.Computers(val.ID==obj.ID)` and make sure also that when calling this command twice, the `Storage` stay single object in the context